### PR TITLE
jira_1234

### PR DESCRIPTION
**Pull request by mananjain02**
[Jira](https://www.google.com)

work done: updated requirenments.txt

reviewers: @mananjain02

### CODE REVIEW

**Comment by healthumbrella** [comment]
changes are not relevant

**Comment by mananjain02** [comment]
we need to have these

File: requirements.txt [hunk 1]
```diff
@@ -42,3 +42,7 @@ tzdata==2023.2
 tzlocal==4.3
 urllib3==1.26.15
 xlwt==1.3.0
+
```
Comment (mananjain02) [inline requirements.txt:45]: this is not appropriate spacing not needed

Comment (healthumbrella) [reply requirements.txt:45]: This can be merged not issues at all

File: requirements.txt [hunk 2]
```diff
@@ -42,3 +42,7 @@ tzdata==2023.2
 tzlocal==4.3
```
Comment (mananjain02) [inline requirements.txt:42]: This version is not needed

Comment (healthumbrella) [reply requirements.txt:42]: we need this

File: requirements.txt (hunk 1 above)
Comment (mananjain02) [reply requirements.txt:45]: okay

**Pull request closed without merging**